internal/services/integrationtesting/consistency_datastore_test.go

```diff
@@ -41,35 +37,34 @@ func TestConsistencyPerDatastore(t *testing.T) {
 				filePath := filePath
 
 				t.Run(path.Base(filePath), func(t *testing.T) {
-					lrequire := require.New(t)
```

Comment:
Because if we try to run for every consistency test file, it takes a VERY long time and it was burning CI minutes. Instead, we use a subset to just ensure that the data access across the different datastore types works